backend/routers/message_routers/show-topic-top.js

```diff
@@ -0,0 +1,24 @@
+const express = require('express');
+
+const router = express.Router();
+const topicService = require('../controllers/message_controllers/topic-service.js');
+
+router.get('/:type', (request, response) => {
```

Comment:
Why not two separate endpoints?

Reply:
globaly its the same functionalyty at the same page (for user)
for example ```www.shazam.com/ru/charts/top-200/world`` and ```https://www.shazam.com/ru/charts/top-200/russia```same thing